app/Models/User.php

```diff
@@ -16,6 +16,7 @@
 use Illuminate\Notifications\Notifiable;
 use Laravel\Passport\HasApiTokens;
 use DB;
+use function GuzzleHttp\Psr7\str;
```

Comment:
I don't think we need this use statement - I believe that str helper is available automagically